db/schema.sql

```diff
@@ -81,13 +81,14 @@ CREATE TABLE users(
 	id INT NOT NULL AUTO_INCREMENT,
 	verified_email BOOLEAN DEFAULT FALSE,
 	-- when inserting into users table the value for email_verification_token should be uuid()
-	email_verification_token VARCHAR(255) NOT NULL,
+	-- Change back to not null when code is further along...
+	email_verification_token VARCHAR(255) DEFAULT NULL,
 	username VARCHAR(30) NOT NULL UNIQUE,
-	first_name VARCHAR(255) NOT NULL,
-	last_name VARCHAR (255) NOT NULL,
-	phone_number VARCHAR(100) NULL,
+	first_name VARCHAR(255) DEFAULT NULL,
```

Comment:
@jace1084 Why are these changed to DEFAULT? I think this is going to create a false database if we run the schema.

Reply:
This is defaulted (for now) due to the dB throwing an error (no default value), therefore I can’t insert other aspects of the signup page unless these values have a default of NULL. This was for testing purposes! I wasn’t planning on merging the the PR yet (that’s why I didn’t assign any reviewers) until I take it out of “testing” mode.

Reply:
ok I see. Interesting. Have you re-run your schema again and see if the app still work fine?